### DISCUSSION TITLE
Figma variables

### DISCUSSION THREAD

**Discussion by margeeta** [material-foundation/material-theme-builder#207] [Q&A]
We'll be releasing an update that includes Figma variables and want to hear your thoughts! 

**Are you using Figma variables? 
If so, are you using modes and how many?**

Modes are limited to certain plans, so we will still be offering themes as styles first, with the option of turning variable generation on.

**Comment by NVLudwig** [comment]
Color variables are great,
It would be a better fit with Material Design System if Figma color variables allowed referencing other colors or properties so we could have the state "brushes" aka "nested color styles" aka "color aliases"  (eg.: PrimaryColor variable + an opacity variable of 8% for Hover States).

It might be worth waiting for Figma to update the variables before releasing something. Switching all the all the Material state brushes, surfaces and tones in Figma when you change the color theme is a giant manual, error-prone pain in the...
